From 9f05429c75bb4fee3501c040e457e7bd69b737c3 Mon Sep 17 00:00:00 2001
From: Tom Englund <tomenglund26@gmail.com>
Date: Fri, 31 Jul 2026 21:47:35 +0200
Subject: [PATCH] drm: use reference in lambda

gotta use reference, otherwise we just drop the lambda SP.
---
 src/backend/drm/DRM.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/backend/drm/DRM.cpp b/src/backend/drm/DRM.cpp
index d18efd3..6be5a71 100644
--- a/src/backend/drm/DRM.cpp
+++ b/src/backend/drm/DRM.cpp
@@ -1582,7 +1582,7 @@ void Aquamarine::SDRMConnector::releaseFBBuffer(const SP<CDRMFB> fb) {
 }
 
 void Aquamarine::SDRMConnector::releaseFBReferences() {
-    const auto releaseFB = [this](SP<CDRMFB> fb) {
+    const auto releaseFB = [this](SP<CDRMFB>& fb) {
         releaseFBBuffer(fb);
         fb.reset();
     };
-- 
2.55.0
